src/components/IconLabelTextLine.tsx

```diff
@@ -35,7 +35,7 @@ const StyledIcon = styled(Icon)`
 
 interface IconLabelTextLineProps {
   dataTestId?: string;
-  icon: string;
+  icon: any;
```

Comment:
?  Denne komponenten blir ikke brukt i denne PR'en. Den burde uansett vært skrevet om for å bruke typografi

Reply:
trenger du å gjøre om fra string til any på denne?